### ISSUE TITLE
Typing and erasing on `iPhoneNumberField` crashing the app

### ISSUE THREAD

**Issue by hesham91fci** [MojtabaHs/iPhoneNumberField#45]
Currently when I press on an `iPhoneNumberField` and erasing the auto added text, things goes well, but one typing just one number and erasing the app crashes throwing infinite messages for `[SwiftUI] Modifying state during view update, this will cause undefined behavior`

The warning is in `iPhoneNumberField.swift` in the following screenshot
![image](https://user-images.githubusercontent.com/4971507/137289501-5112fb15-e402-462f-8936-5332be7c5c69.png)

This is the part which I'm creating `iPhoneNumberField`

```
var numberField: some View {
        iPhoneNumberField("Phone", text: $mobileNumber)
            .font(UbixFont.light.uiFont)
            .flagHidden(false)
            .flagSelectable(false)
            .prefixHidden(false)
            .autofillPrefix(true)
            .cornerRadius(10)
            .padding(.vertical, 8)
            .padding(.horizontal, 6)
            .overlay(RoundedRectangle(cornerRadius: 6, style: .continuous).stroke(textfieldBorderColor, lineWidth: 1))
            .padding(.vertical, 0)
    }

```
Any help would be appreciated

**Comment by mylogon341** [comment]
I have just witnessed this too. Any suggestions? Did you solve it or replace this lib out for something else?

**Comment by MojtabaHs** [comment]
Merged in 0.9.0. Thanks to Sega-Zero
